timetagger_cli/core.py

```diff
@@ -85,6 +89,8 @@ def setup(args=None):
     filename = prepare_config_file()
     print("Config file: " + filename)
     print("Will now (try to) open the config file. Just edit and save the file.")
+    # Config file contains user secret so it should not be readable to others
+    os.chmod(filename, 0o640)
```

Comment:
Would be cleaner to do this where the file is created, in `prepare_config_file()`

Reply:
What should happen if the config file already exists and the user wants to setup the config again? Should it overwrite the permissions just in case the user have messed with them or leave the permissions unchanged and potentially exposing the user's secret?

Reply:
I guess the file can be kindof considered an "implementation detail", so let's set it every time. What do you think?

Reply:
I think it would be better to check if the existing file is a config file and try to mess less with user files in case it just so happens the user have some important, non-program related information.

I can commit now with that fix, but in the future it would be better to add some additional check, including what if the path is actually a directory and not a file.